finance/accounting.md

```diff
@@ -1,52 +1,68 @@
+{term}`CS&S` provides us a monthly dataset of the last 6 months of all of all our accounting transactions.
+This is [in a Google Sheet that is automatically updated each day][gsheet].
+It includes the last 6 months of any transaction 2i2c has made (if we need more than 6 months of history, ask `fsp@2i2c.org` for a custom data dump).
 
-- [2i2c Drive Folder with our finance assets](https://drive.google.com/drive/folders/1D5NQKhPDP6zMQ8EdLcMOceTz-ek81nmX?usp=sharing) contains 2i2c-specific budgeting and modeling information. It is available to all team members.
+We have a copy of this sheet in an [**Accounting Transactions AirTable**][accounting-table][^airtable].
+Any columns that are synchronized with CS&S have a little lightning bolt (⚡) next to them.
+We use this to create [our accounting dashboard](accounting:dashboard).
+
+[^airtable]: See [](../administration/airtable.md) for how to access AirTable.
+
+[gsheet]: https://docs.google.com/spreadsheets/d/1qH5IK18z79X8cEwlwDwnlArTiYnbRVIPGqdOUdrsF0c/edit?usp=sharing
+
+[accounting-table]: https://airtable.com/appbjBTRIbgRiElkr/tblDKGQFU0iEIa5Qb/viwAdsIgMwbqKDdZ0
+
+### Update our accounting data airtable
+
+We have a semi-automated process to update the data in our accounting table.
+Here are the steps to follow:
+
+% TODO: We should automate this with a CRON job, make.com/zapier integration, etc.
+%   ref: https://github.com/2i2c-org/team-compass/issues/703
+1. **Confirm that [the Google Sheet][gsheet] has been updated** by checking the `Automatic Operations Events Log` tab.
+2. **Delete all of the records** in the [accounting table][accounting-table].
+3. **Import the google sheets data**. Go to `Accounting Transactions` -> `Import Data` -> `Google Sheets` -> `Google sheets account`.
+   - Here's an image of the menu you want:
+
+     ![image](https://user-images.githubusercontent.com/1839645/230121196-0d398812-ba22-4cea-a42f-e3ad644a3e19.png)
+   - You'll see a list of Google Sheets in our account. Import the one titled `2i2c FYE23 Account Transactions - Auto Generated`.
+   - Check the `Skip first row` option during the import, so that we don't import the section title names.
+
+This should automatically import the data into the proper columns, you don't need to do anything else.
 
-- [CS&S Drive Folder with their finance assets](https://drive.google.com/drive/folders/115EIa6cD4BNGIqOd2i7Rqu3MgsM73lgR?usp=sharing) contains assets that are stewarded by CS&S. It is available to 2i2c'c executive director.
+When this action happens, an [AirTable automation will run](https://airtable.com/appbjBTRIbgRiElkr/wflVJQz277S6lF0E3/wtrHzwIWJLGTnJl0m) and attempt to link each new record with a corresponding **Contract**, using the `Xero ID` field.
+If a contract with the record's `Xero ID` is found, then a linked record will be created in the `Contracts` column.
 
-See these subsections below for more information about accounting data.
+Here's a brief video describing the above process.
```

Comment:
I had to authenticate inside the embedded frame to get access to the video. After clicking, I was notified that access was granted. The video spun and spun inside the frame but rendered properly when I clicked the pop out arrow. 

The video is super helpful! That said, I noticed a few issues. The tabs at the bottom of the Google doc at the beginning are not visible in the video so the narration pointing out that a record of updates is available in another tab confused me at first. There is a grey bar blocking the top of the AirTable at 1:12.

Reply:
ah shit I think that's Zoom "blacking out" the areas where its toolbar is. I need to figure out how to better-record zoom videos. Will try to do another recording tomorrow.